### DISCUSSION TITLE
Merge-PnPTerm not recognized as cmdlet

### DISCUSSION THREAD

**Discussion by FloWaeger** [pnp/powershell#3889] [Q&A]
Hello Community,

I tried to merge many Terms via PnP PS with cmdlet Merge-PnPTerm but error message says it is not recognized.

https://pnp.github.io/powershell/cmdlets/Merge-PnPTerm.html 

Any ideas? We talk about more than 200 Terms to merge so I need a script based solution.

BR
Florian

**Comment by AndersRask** [comment]
Please show us some code how you try to do it.
Also try to verify that the GUIDs are correct. You can find them in the UI too, under "Usage settings"
Check weather you are using global or site level term groups.

**Reply by FloWaeger** [reply]
After an update to current PS and PnP.PowerShell version, everything worked.
